### ISSUE TITLE
How to allow an empty value?

### ISSUE THREAD

**Issue by dspangenberg** [roboncode/orango#21]
I can't save my document if a field is empty. Have tried some options, without sucess:

`[Joi.string().empty() ` => #19 

```
firstName: {
  type: 'string.empty'
}
``` => #20 

```
firstName: {
  type: 'string.empty',
  optional: true
}
``` => #20

**Comment by roboncode** [comment]
Try this...
```js
{
   firstName: String
}
```

**Comment by roboncode** [comment]
Download master and see if this works for you. I tested it. There may be changes that are not yet available until the next release that may have resolved this issue.

```js
await orango.connect()

const UserSchema = orango.Schema({
  name: String
})

let User = await orango.model('User', UserSchema)
let user = await new User().save()
console.log('User created', user)
```

**Comment by roboncode** [comment]
This fix has been released in version 0.7.0
